### PR TITLE
docs: migrating csi - add info to ensure that the cluster and storage solutions are health prior migrations

### PR DESCRIPTION
**Pull request by camilamacedo86**
… 

To add info to ensure that the cluster and storage solutions are health prior migrations

### CODE REVIEW

File: src/markdown-pages/install-with-kurl/migrating-csi.md [hunk 1]
```diff
@@ -74,6 +74,11 @@ Before you attempt to change the CSI provisioner in your cluster, complete the f
 
 - The add-ons Kots and Registry require an Object Storage API (similar to AWS S3) to be available in the cluster. This API can be provided either by Rook (directly) or by the OpenEBS+Minio duo (OpenEBS provides the storage and Minio the Object Storage API). If you are using Kots or Registry add-ons in your cluster then the available migration paths are to migrate to Rook or to OpenEBS+Minio.
 
+- Ensure that cluster is a healthy state prior do any migration. Check if the Nodes are Ready and if all Pods are running. Also, if you are migrating from an storage solution to another please ensure that it is healthy prior the migration. Beyond check if all Pod(s) into this namespace are Running you might want check the storage state:
```
Comment (camilamacedo86) [inline src/markdown-pages/install-with-kurl/migrating-csi.md:77]:
```suggestion
- Ensure that cluster is a healthy state prior to perform any migration. Check if the Nodes are Ready and if all Pods are running. Also, if you are migrating from an storage solution to another please ensure that it is healthy. To do perform this checks see:
```

File: src/markdown-pages/install-with-kurl/migrating-csi.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Before you attempt to change the CSI provisioner in your cluster, complete the f
 
 - The add-ons Kots and Registry require an Object Storage API (similar to AWS S3) to be available in the cluster. This API can be provided either by Rook (directly) or by the OpenEBS+Minio duo (OpenEBS provides the storage and Minio the Object Storage API). If you are using Kots or Registry add-ons in your cluster then the available migration paths are to migrate to Rook or to OpenEBS+Minio.
 
-- Ensure that cluster is a healthy state prior do any migration. Check if the Nodes are Ready and if all Pods are running. Also, if you are migrating from an storage solution to another please ensure that it is healthy prior the migration. Beyond check if all Pod(s) into this namespace are Running you might want check the storage state:
+- Ensure that cluster is a healthy state prior to perform any migration. Check if the Nodes are Ready and if all Pods are running. Also, if you are migrating from an storage solution to another please ensure that it is healthy. To do perform this checks see:
```
Comment (rrpolanco) [inline src/markdown-pages/install-with-kurl/migrating-csi.md:77]:
```suggestion
- Ensure that cluster is a healthy state prior to perform any migration. Check if the Nodes are Ready and if all Pods are running. Also, if you are migrating from an storage solution to another please ensure that it is healthy. To perform these checks see:
```

File: src/markdown-pages/install-with-kurl/migrating-csi.md [hunk 3]
```diff
@@ -74,6 +74,11 @@ Before you attempt to change the CSI provisioner in your cluster, complete the f
 
 - The add-ons Kots and Registry require an Object Storage API (similar to AWS S3) to be available in the cluster. This API can be provided either by Rook (directly) or by the OpenEBS+Minio duo (OpenEBS provides the storage and Minio the Object Storage API). If you are using Kots or Registry add-ons in your cluster then the available migration paths are to migrate to Rook or to OpenEBS+Minio.
 
+- Ensure that cluster is a healthy state prior to perform any migration. Check if the Nodes are Ready and if all Pods are running. Also, if you are migrating from an storage solution to another please ensure that it is healthy. To perform these checks see:
```
Comment (rrpolanco) [inline src/markdown-pages/install-with-kurl/migrating-csi.md:77]:
```suggestion
- Ensure that cluster is in a healthy state prior to performing any migration. Check if the nodes are Ready and if all Pods are running. Also, if you are migrating from a storage solution to another please ensure that it is healthy. To perform these checks see:
```

**Comment by rrpolanco** [comment]
> don't we already have preflights for this?

yeah, we do. this is why I think this change does not need to be documented.

**Comment by rrpolanco** [comment]
Documentation not warranted given that the installer already performs CSI migration checks and checks the health of the cluster in its host preflight phase.

**Pull request closed without merging**